EmployeeManagement/Tavisca.EmployeeManagementService.Implementaion/EMSPostImplementation.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Tavisca.EmployeeManagement.DataContract;
+using Tavisca.EmployeeManagement.ServiceContract;
+using Tavisca.EmployeeManagement.Translator;
+using Tavisca.WCF.Model;
+namespace Tavisca.EmployeeManagementService.Implementaion
+{
+    public class EMSPostImplementation:IEmployeeManagementService
+    {
+       public Employee Create(Employee employee)
+        {
+            EmployeeModel employeeModel = Translator.EmployeeModelGenerator(employee);
+            string id = employeeModel.GenerateId();
```

Comment:
No need to have a GenerateId method, just assign the GUID in the manager or the class constructor.